### PR TITLE
show server port to users after servers starts

### PR DESCRIPTION
**Pull request by omidfi**

### CODE REVIEW

File: src/bin/server.js [hunk 1]
```diff
@@ -3,8 +3,9 @@ const config = require('../config')
 const app = require('../app')
 
 process.once('SIGINT', shutDown)
-process.once('SIGTERM', shutDown)
+process.once('SIGTERM', shutDown) 
 
+console.log('Starting the server on: http(s)://', config.server.host, ': ', config.server.port)
 app.server.listen(config.server.port, config.server.host)
```
Comment (dimonnwc3) [inline src/bin/server.js]: great! and I have one small suggestion.
to ensure, that message appears only after the server starts, we need to move it inside asynchronous [listen callback](https://nodejs.org/dist/latest-v7.x/docs/api/http.html#http_server_listen_port_hostname_backlog_callback)

in this case, a user will see either server start successfully or some error like `EADDRINUSE ` happens

Comment (omidfi) [inline src/bin/server.js]: @dimonnwc3 good point. 
I was just too lazy to do it :D 

I hope it's fine now.

File: src/bin/server.js [hunk 2]
```diff
@@ -18,3 +19,11 @@ function shutDown () {
     })
   }
 }
+
+// let the user know if starting the server failed
+function status(error) {
+  if(error) {
```
Comment (dimonnwc3) [inline src/bin/server.js]: there is no error argument
it's just success callback for the listening event: https://nodejs.org/dist/latest-v7.x/docs/api/net.html#net_event_listening

error throws already automatically to console, so we don't need to care about it

so only success message will be enough

**Comment by omidfi** [comment]
I hope this is fixed now :) 
I've never used the raw node server. This was a good exercise.

**Comment by dimonnwc3** [comment]
@omidfi looks good

**Pull request merged**